openqa-trigger-bisect-jobs

```diff
@@ -299,10 +299,19 @@ def main(args):
             ]
         )
 
-        out = openqa_clone(
-            params,
-            args.dry_run,
-        )
+        try:
+            out = openqa_clone(
+                params,
+                args.dry_run,
+            )
+        except subprocess.SubprocessError as err:
+            p_stderr = err.output if err.output else ""
+            log.error(f"{type(err).__name__}: {p_stderr} ")
+            if 'the repositories for the below updates are unavailable' not in str(err.output):
+                sys.exit(err.returncode)
```

Comment:
```suggestion
                raise err
```

Reply:
this prints again a stack trace. why?

Reply:
But i know all that. I am just focused to the fact that the ticket was about no stack trace. And what we have done is to capture the SubprocessError and log it. if we need to track other problem, it would make sense to create custom Exceptions. But I avoided to go this route because I knew i will have other problems with that.